insights/tests/datasources/test_yum_updates.py

```diff
@@ -0,0 +1,10 @@
+from insights.core.spec_factory import DatasourceProvider
+from insights.specs.datasources.yum_updates import yum_updates
+
+
+# Verify that the yum_updates broker correctly executes.
+def test_yum_updates_runs_correctly():
+    broker = {}
+    result = yum_updates(broker)
```

Comment:
This doesn't test that any of the data is processed correctly right? Would it be possible to add in tests based on mocked output?

Reply:
Ok, updated with a mocked data test.

Reply:
Awesome thank you.